crashlog/crashutils.h

```diff
@@ -66,6 +66,7 @@ int do_screenshot_copy(char* bz_description, char* bzdir);
 
 void do_wdt_log_copy(char *dir);
 void do_last_kmsg_copy(char *dir);
+static int find_system_last_kmsg();
```

Comment:
If we declare this function as static directly in crashutils.c, then we don't need to this line in header file.